webviz_subsurface/plugins/_reservoir_simulation_timeseries_onebyone.py

```diff
@@ -476,3 +550,8 @@ def filter_ensemble(data, ensemble, vector):
 @webvizstore
 def read_csv(csv_file) -> pd.DataFrame:
     return pd.read_csv(csv_file, index_col=None)
+
+
+@CACHE.memoize(timeout=CACHE.TIMEOUT)
+def get_unit(smry_meta, vec):
+    return "" if smry_meta is None else simulation_unit_reformat(smry_meta.unit[vec])
```

Comment:
Would it be simpler/better that this returns `None` when there is no unit, instead of empty string?

Reply:
It is probably "better", and was considering it, but empty string is "safer" in the sense that no errors are thrown if you don't include `None` handling each place the `get_unit` is used. Quite easy to change it still, so I can if you want me to.

Reply:
Vote for changing to `None`. Then I guess it would also be easy to distinguish between missing/undefined unit, and a unit-less quantity/ratio?

Reply:
👍 will do. And yes, that should be easy.